measure/src/main/scala/org/apache/griffin/measure/Application.scala

```diff
@@ -104,12 +106,18 @@ object Application extends Loggable {
       case Success(_) =>
         info("process end success")
       case Failure(ex) =>
-        error(s"process end error: ${ex.getMessage}")
+        error(s"process end error: ${ex.getMessage}", ex)
         shutdown
         sys.exit(-5)
     }
 
     shutdown
+
+    if (success) {
```

Comment:
looks like this one breaks a lot of stuff

Reply:
removed exit(0), looks like org.apache.spark.deploy.yarn.ApplicationMaster is not liking direct exit(0)